manifests/install.pp

```diff
@@ -60,8 +60,7 @@
       group              => $wso2_group,
       install_dir        => $install_dir,
       pack_dir           => $pack_dir,
-      carbon_home        => $carbon_home,
-      notify             => Wso2base::Ensure_directory_structures[$install_dirs]
+      require            => Wso2base::Ensure_directory_structures[$install_dirs]
```

Comment:
Not sure if this dependency is needed. It might lead to clean and file copy being executed parallely.

Reply:
According to [1] execution will be ensure_directory_structures -> clean_deployment

[1] https://docs.puppet.com/puppet/5.1/lang_relationships.html#syntax-relationship-metaparameters

Reply:
Yes, but shouldn't it be the other way. Clean first, then create directory structures?

Reply:
Changed as requested